appland/popup/record.js

```diff
@@ -1,23 +1,49 @@
+const chrome = (window.chrome || window.browser);
```

Comment:
AFAIK, all the browsers we might want to run this extension (Chrome, Firefox, Edge) have `window.chrome`, so this is effectively a no-op. Is there a reason to keep it?

I changed the name so it matches the behavior of the thing it's shadowing. (In Firefox, `browser` is used to indicate that any function called from it will return a `Promise`, rather than using a callback.)

Reply:
I don't remember the details on this, I think it's a pre-chrome normalization for Edge. Given that, it's probably not important we keep it.

Reply:
In fact this causes the extension to break in Chrome:
```
Uncaught SyntaxError: Identifier 'chrome' has already been declared
```